android/app/src/main/java/com/reactnativenavigation/modal/RnnModal.java

```diff
@@ -66,11 +73,18 @@ public void onDisplayed() {
     }
 
     public void push(Screen screen) {
+        updateToolbar(screen, mToolBar);
```

Comment:
It didn't look as though we were previously updating the toolbar style for a pushed screen within a modal.